### ISSUE TITLE
Transifex is broken beyond repair on every level, change to Pootle or Weblate.

### ISSUE THREAD

**Issue by comradekingu** [rakekniven/Transifex-Issues#19]
I have used Transifex heavily almost every day for 5 years. I still discover new bugs, daily.
Show me any function, and i will tell you how it is broken.
In addition to that, the upstream developers and support team does not listen, even if you pay through the nose for it. It is the worst joke in software to date.

[Pootle](https://pootle.locamotion.org) and [Weblate](https://hosted.weblate.org/) have built in checks making for fewer errors in the final product. It also checks for consistency, which if you want some level of on Transifex, you have to search yourself to death, in the broken search. Battling with suggestions from deleted strings (!)
There is no fuzziness, there is no link to upstream, can't share links to anyone, cant see the strings, it is non-free, loads every spying script under the sun, etc etc etc.

I propose solving the issues present on Transifex now, looking into the ones filed against owncloud too, and then moving.

**Comment by rakekniven** [comment]
Allan, your postings are not helpful at all here. Please contact project maintainters

**Comment by comradekingu** [comment]
I am puzzled as to how they could ever be, were I or those that have said the same, to tell you anything else.

My issue with this effort is it is in stark contrast to the wishes of every other contributor voicing the exact same concern. Not that it isn't a laudable, however intangible goal, that you are free to chase. The Nextcloud contributors want to move away from Transifex, not amend it. It fails for the exact same reason for every other project. What exactly are we doing here that hasn't failed before, in light of better and compatible alternatives?

The effort at best bridges a gap into uncanny valley. It didn't work for TX when they were doing it, and they have since closed off the core. Perhaps for reasons not airing bug-reports sitting around forever garnering bad publicity.

You are not the first person to reach out to TX. As is evident from the feedback thus far.

Furthermore, I can only think this isn't compatible with the gratis guidelines of GitHub, as there is no code attached to it. My concern extends to time and effort better spent elsewhere.

So to sum it up, they know, the didn't do anything about it, other than the exact opposite, removing all ability for recourse, while continuing in their ways.

And here we are, trying to shake a stick at the most buggy thing since Internet Explorer.

**Comment by wetneb** [comment]
I wholeheartedly recommend Weblate to any localized open source project.
